packages/web/src/hooks/useAppAuthClients.ee.ts

```diff
@@ -6,25 +6,27 @@ import { GET_APP_AUTH_CLIENTS } from 'graphql/queries/get-app-auth-clients.ee';
 
 type QueryResponse = {
   getAppAuthClients: AppAuthClient[];
-}
+};
 
-export default function useAppAuthClient(appKey: string) {
-  const [
-    getAppAuthClients,
+export default function useAppAuthClient(appKey: string, active?: boolean) {
```

Comment:
`useAppAuthClient('gitlab', { active: true })`

or

```
useAppAuthClient({
  appKey: 'gitlab',
  active: true
});
```

may be better for readability.